### PR TITLE
Upgrading Gradle to 6.9.2, Kotlin to 1.4.20, junit to 4.13.2 and mockito to 4.4.0

### PR DESCRIPTION
**Pull request by vetler**
Upgrading Gradle to 6.9.2, Kotlin to 1.4.20, junit to 4.13.2 and mockito to 4.4.0.

### CODE REVIEW

File: mockito-kotlin/build.gradle [hunk 1]
```diff
@@ -23,9 +23,9 @@ dependencies {
     compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compileOnly 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0'
 
-    compile "org.mockito:mockito-core:4.0.0"
+    compile 'org.mockito:mockito-core:4.4.0'
```
[the remark this answers is not in the document]
Comment (vetler) [reply mockito-kotlin/build.gradle:26]: Absolutely, I'll try that when it's released

[the remark this answers is not in the document]
Comment (TimvdLippe) [reply mockito-kotlin/build.gradle:26]: https://repo1.maven.org/maven2/org/mockito/mockito-core/4.5.1/ has been released 😄

**Comment by vetler** [comment]
@TimvdLippe :white_check_mark:

**Review by TimvdLippe — APPROVED** [review]
Thanks!

**Pull request merged**